### ISSUE TITLE
"New [School/Department/Course/Section" not keyboard accessible

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs-admin#188]
In the "Schools" view, the "New School" link cannot be accessed using only the keyboard.

**Comment by Malorn44** [comment]
mine
